hack/shellcheck.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -eux
+
+IS_CONTAINER="${IS_CONTAINER:-false}"
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
+
+if [ "${IS_CONTAINER}" != "false" ]; then
+    TOP_DIR="${1:-.}"
+    find "${TOP_DIR}" \( -name '*.sh' -o -wholename 'scripts/*' \) -exec shellcheck -s bash {} \+
```

Comment:
Not sure if we need/should have this `-o -wholename 'scripts/*'`. I think it means that it will check files that end with `.sh` or files located in `scripts` no matter what name they have.